feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.ts

```diff
@@ -27,32 +29,37 @@ export class ConfiguratorTextfieldService {
    *
    * @returns {Observable<ConfiguratorTextfield.Configuration>}
    */
-  public createConfiguration(
+  createConfiguration(
     owner: GenericConfigurator.Owner
   ): Observable<ConfiguratorTextfield.Configuration> {
     this.store.dispatch(
-      new ConfiguratorActions.CreateConfiguration({
+      new ConfiguratorTextfieldActions.CreateConfiguration({
         productCode: owner.id, //owner Id is the product code in this case
         owner: owner,
       })
     );
 
-    return this.store.select(ConfiguratorSelectors.getConfigurationContent);
+    return this.store.pipe(
+      select(ConfiguratorTextFieldSelectors.getConfigurationContent)
+    );
   }
 
   /**
-   * Updates a textfiled configuration, specified by the changed attribute.
+   * Updates a textfield configuration, specified by the changed attribute.
    *
    * @param changedAttribute - Changed attribute
    */
-  public updateConfiguration(
+  updateConfiguration(
```

Comment:
Hmm, even though you're using `take(1)`, is subscribing absolutely necessary?

Reply:
The only way around this seems to pass the configuration to the method (which would make the components a bit more complex).
Is this bad practise (after take(1) I thought it's ok?)